var/spack/repos/builtin/packages/mofem-users-modules/package.py

```diff
@@ -0,0 +1,101 @@
+##############################################################################
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+from distutils.dir_util import copy_tree
+import os
+
+
+class MofemUsersModules(CMakePackage):
+    """mofem users modules"""
+
+    homepage = "http://mofem.eng.gla.ac.uk"
+    version('1.0', '5a8b22c9cdcad7bbad92b1590d55edb1', expand=False)
+    url = "https://bitbucket.org/likask/mofem-joseph/downloads/users_modules_dummy"
+
+    maintainers = ['likask']
+
+    variant('copy_user_modules', default=True,
+        description='Copy user modules directory instead linking')
+    variant('with_metaio', default=False,
+        description='Install MetaIO with MoFEM users modules')
+
+    variant('mofem-fracture-module', default=False,
+        description="Install fracture mechanics module")
+    variant('mofem-minimal-surface-equation', default=False,
+        description="Install minimal surface equation module")
```

Comment:
Variants should avoid the use of dashes. The parser won't have any trouble, but to a human user, `-mofem-minimal-surface-equation` is ambiguous as to whether you are disabling 1 variant or 4.

Reply:
Agree, I don't like as well. Do you suggest underscore, would be better?

Reply:
Yeah, I usually use underscores.